esx_service/volume_kv.py

```diff
@@ -82,7 +84,7 @@
 ACCESS_TYPES = [ACCESS_READWRITE, ACCESS_READONLY]
 
 # Filesystem type
-# This option is handled in the volume-plugin at the docker host, and tracked in volume metadata. 
+# This option is handled in the volume-plugin at the docker host, and tracked in volume metadata.
```

Comment:
Hi,

FWIW, I attempted to save this file several times, even outside my IDE (Atom) via `vim`, but every save attempt marked this line as changed. It appears the editors, even `vim`, were stripping the trailing whitespace character.